cli/cmd/root.go

```diff
@@ -42,6 +45,11 @@ func NewCmdRoot() *cobra.Command {
 	noLogFile := false
 	logFileName := "cloudquery.log"
 	sentryDsn := sentryDsnDefault
+	var err error
+	if invocationUUID.UUID, err = guuid.NewRandom(); err != nil {
+		fmt.Fprintf(os.Stderr, "failed to generate invocation uuid: "+err.Error())
```

Comment:
If you don’t do %s, it will treat any potential % chars in the error as placeholders and cause trouble.

Reply:
Fixed in [2e15630](https://github.com/cloudquery/cloudquery/pull/18878/commits/2e15630916c63adf288558fc7f685589b2da8680)